Add pricing note to topline docs page directly (#24979)
sourcegraph/sourcegraph: doc/code_insights/index.md

# Code Insights <a href="../admin/beta_and_experimental_features"><sup><span class="badge badge-beta">Beta</span></sup></a>

> 

Reviewer: ```suggestion
Code Insights are the analytics tool that tells you things about your codebase at a high level based on how your code changes over time.

Code Insights are based on our universal code search, making them incredibly accurate and extremely configurable. Track anything that can be expressed with Sourcegraph search query: migrations, package use, version adoption, code smells, codebase size and much more, across 1,000s of repositories.
```

Reviewer: I'd add some intro text here aligned with the popup we're working on.

Author: Sure! Definitely we want more context on this page/more of a "designed" feel to the page as we add more docs, similar to other top-level docs pages; this is a good start.

> > NOTE: While in Beta, Code Insights is free for enterprise customers. Once Code Insights is generally available and no sooner than January 1, 2022, it may cost an additional amount. 

Reviewer: ```suggestion
> NOTE: While in Beta, Code Insights is free for enterprise customers. Once Code Insights is generally available and no sooner than January 1, 2022, to continue using Code Insights may require a separate paid plan.
```

Reviewer: @Joelkw aligning with the popup copy, it sounds more thoughtful imo

Author: Sounds good!

Code Insights are currently in Private [Beta](../admin/beta_and_experimental_features.md). If you're a customer who wants early access or needs support, please [send us an email](mailto:feedback@sourcegraph.com).

Code insights docs are actively in progress during the beta.

> 

Reviewer: ```suggestion
It means that we're still polishing Insights and you might find bugs while we’re in beta. Please, do [share any bugs 🐛 or feedback](mailto:feedback@sourcegraph.com) to help us make Code Insights better.
```

Reviewer: @Joelkw adding info about feedback, we can link this place from the popup.

# Code Insights <a href="../admin/beta_and_experimental_features"><sup><span class="badge badge-beta">Beta</span></sup></a>

> Code Insights are the analytics tool that tells you things about your codebase at a high level based on how your code changes over time.

Author: ```suggestion
Code Insights tells you things about your codebase at a high level, based on both how your code changes over time and its current state.
```

> Code Insights are based on our universal code search, making them incredibly accurate and extremely configurable. Track anything that can be expressed with Sourcegraph search query: migrations, package use, version adoption, code smells, codebase size and much more, across 1,000s of repositories.

Author: ```suggestion
Code Insights is based on our universal code search, making it incredibly precise and extremely configurable. Track anything that can be expressed with Sourcegraph search query: migrations, package use, version adoption, code smells, codebase size and much more, across 1,000s of repositories.
```

> NOTE: While in Beta, Code Insights is free for enterprise customers. Once Code Insights is generally available and no sooner than January 1, 2022, to continue using Code Insights may require a separate paid plan.

Code Insights are currently in Private [Beta](../admin/beta_and_experimental_features.md). If you're a customer who wants early access or needs support, please [send us an email](mailto:feedback@sourcegraph.com).

Code insights docs are actively in progress during the beta.

> It means that we're still polishing Insights and you might find bugs while we’re in beta. Please, do [share any bugs 🐛 or feedback](mailto:feedback@sourcegraph.com) to help us make Code Insights better.

Author: ```suggestion
We're still polishing Code Insights and you might find bugs while we’re in beta. Please [share any bugs 🐛 or feedback](mailto:feedback@sourcegraph.com) you have to help us make Code Insights better.
```
